packages/core/src/collection.ts

```diff
@@ -1,7 +1,9 @@
 import A from 'automerge'
 import { DELETE_COLLECTION } from './constants'
-import { docSetToObject } from './docSetHelpers'
-import { ChangeMap, DocSetState } from './types'
+import { ChangeMap, RepoSnapshot } from './types'
+import { Repo } from './Repo'
+
+export const DELETED = '::DELETED'
```

Comment:
I wonder if this is the right location for this constant. I'd probably move it to Repo (w/e)

Reply:
I've moved it to constants.ts
70e72ab275d7791c94ce24fd2ccaf785cc437aca